docs/notes/vineyard-operator.rst

```diff
@@ -97,7 +79,7 @@ Install from source code
 
     $ cd k8s
     $ make -C k8s docker-build
-    $ kind load docker-image registry-vpc.cn-hongkong.aliyuncs.com/libvineyard/vineyard-operator:latest
+    $ kind load docker-image vineyardcloudnative/vineyard-operator:latest
```

Comment:
The default dev image (built from the code) should be `registry-vpc.cn-hongkong.aliyuncs.com/libvineyard/vineyard-operator:latest`

Reply:
Refer https://github.com/v6d-io/v6d/blob/ac714de200ced3daaa72bdca9b18f5e86bdfbc4a/k8s/test/e2e/vineyardd/e2e.yaml#L31-L34